### PR TITLE
feat: event emitter that works in browser

### PR DESCRIPTION
**Pull request by pavanjoshi914**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3506,6 +3506,11 @@ emittery@^0.13.1:
   resolved "https://registry.npmjs.org/emittery/-/emittery-0.13.1.tgz"
   integrity sha512-DeWwawk6r5yR9jFgnDKYt4sLS0LmHJJi3ZOnb5/JdbYwj3nW+FxQnHIjhBKz8YLC7oRNPVM9NQ47I3CVx34eqQ==
 
+emittery@^1.0.3:
```
[the remark this answers is not in the document]
Comment (pavanjoshi914) [reply yarn.lock:3509]: i removed package using yarn remove eventemitter3. this makes changes in package.json as well as yarn.lock. but there were no changes in yarn.lock. am I missing something?

[the remark this answers is not in the document]
Comment (rolznz) [reply yarn.lock:3509]: Yes, because there is eventemitter3 in my yarn.lock

[the remark this answers is not in the document]
Comment (rolznz) [reply yarn.lock:3509]: Ah ok, thanks!

**Comment by bumi** [comment]
Does it work with the simple-boost button?

**Review by rolznz — APPROVED** [review]
utACK

**Comment by pavanjoshi914** [comment]
it does work. 

image


also crossverified that I did yarn link correctly (I can see js SDK node_modules using emittery	after yarn link
) so shall be good to go. we need to update js-sdk in bitcoin-connect and use new version of bitcoin-connect in simple boost 

image

**Comment by bumi** [comment]
can you push the simple-boost with a git dependency to this one?

**Comment by rolznz** [comment]
@pavanjoshi914 can you fix the conflict and merge it?

**Comment by pavanjoshi914** [comment]
done

**Pull request merged**